cime_config/testdefs/testlist_clm.xml

```diff
@@ -540,35 +540,17 @@
   </test>
   <test name="ERS_Lm13" grid="f10_f10_mg37" compset="I1850Clm51Bgc" testmods="clm/monthly_matrixcn_fast_spinup">
     <machines>
-      <machine name="cheyenne" compiler="intel" category="aux_clm"/>
+      <machine name="derecho" compiler="intel" category="aux_clm"/>
       <machine name="izumi"    compiler="intel" category="aux_clm"/>
     </machines>
     <options>
       <option name="wallclock">01:20:00</option>
       <option name="comment"  >Run matrix solution fast spinup for restart beyond a year (to trigger next year logic) turning off extra options to make faster (so without crop or isotopes). This test would've caught a previous bug.</option>
     </options>
   </test>
-  <test name="SMS_Lm1" grid="f10_f10_mg37" compset="I1850Clm45Bgc" testmods="clm/clm45_monthly_matrixcn_soilCN30">
```

Comment:
@ekluzek we started discussing this test but didn't conclude. 
Here's a plan that I'm open to revising.

- [x] Eliminate this test because it's a clm45 test and as such requires a modified clm45 params file as you can see here `/glade/work/slevis/git/cn-matrix_v3/cime_config/testdefs/testmods_dirs/clm/clm45_monthly_matrixcn_soilCN30/user_nl_clm`
- [ ] Try it as a 5.1 test. If it fails, open an issue and bring it back as a 5.1 test.
- [ ] Replace the clm45_monthly_matrixcn_soilCN30 testmods directory with a 5.1 version.

Let me know what you think.

Reply:
From what I can tell this isn't tied to being clm4_5. This is just trying out CN ratios for soil that are much larger than the default (which are around 8 to 11) to 30.

The parameter file has changed since this test was created, but you could change bgc_cn_s1,bgc_cn_s2,bgc_cn_s3 to 30 for the ctsm51 parameter file for this test case.